examples/documentation/CMakeLists.txt

```diff
@@ -32,3 +33,10 @@ foreach(example_program ${example_programs})
     pika_add_example_test("documentation" ${example_program} ${${example_program}_PARAMETERS})
   endif()
 endforeach()
+
+if(PIKA_WITH_TESTS AND PIKA_WITH_TESTS_EXAMPLES)
+  set_tests_properties(
+    tests.examples.documentation.require_started_documentation PROPERTIES PASS_REGULAR_EXPRESSION
+                                                                          "terminate called"
```

Comment:
libc++ prints a different message: https://github.com/pika-org/pika/actions/runs/11779938152/job/32809450059#step:7:410. Perhaps change this to match ` pika::execution::experimental::~require_started_sender: A require_started sender was never started` instead, since that's a message we control?